src/ansys/fluent/core/session_pure_meshing.py

```diff
@@ -109,13 +109,13 @@ def workflow(self):
 
     def watertight(self, dynamic_interface=True):
         """Get a new watertight workflow."""
-        self._base_meshing.new_workflow.watertight(dynamic_interface)
-        return self._base_meshing.new_workflow
+        self._base_meshing.new_wt_workflow.watertight(dynamic_interface)
+        return self._base_meshing.new_wt_workflow
 
     def fault_tolerant(self, dynamic_interface=True):
         """Get a new fault-tolerant workflow."""
-        self._base_meshing.new_workflow.fault_tolerant(dynamic_interface)
-        return self._base_meshing.new_workflow
+        self._base_meshing.new_ft_workflow.fault_tolerant(dynamic_interface)
+        return self._base_meshing.new_ft_workflow
```

Comment:
is all this code here because the other meshing mode inherits from it?

Reply:
yes

Reply:
OK good